.docker/entrypoint/spec/jekyll_server_spec.rb

```diff
@@ -18,37 +18,58 @@
   end
 
   describe '#execute :serve' do
-    describe 'weird file' do
-      data_dir = File.join(__dir__, '..', '..', '..', 'tests', 'full')
+    before(:all) do
+      @thread = Thread.new do
+        context = Context.new('development', __dir__, data_dir)
+        jekyll_config_provider = JekyllConfigProvider.new(context)
+        context.configuration = jekyll_config_provider.provide('serve')
+        jekyll_server = JekyllServer.new(context)
+        jekyll_server.execute
+      end
+      @thread.abort_on_exception = true
 
-      before(:all) do
-        @thread = Thread.new do
-          context = Context.new('development', __dir__, data_dir)
-          jekyll_config_provider = JekyllConfigProvider.new(context)
-          context.configuration = jekyll_config_provider.provide('serve')
-          jekyll_server = JekyllServer.new(context)
-          jekyll_server.execute
-        end
-        @thread.abort_on_exception = true
-
-        JekyllServe.mutex.synchronize do
-          running = JekyllServe.running?
-          JekyllServe.run_cond.wait(JekyllServe.mutex) unless running
-        end
+      JekyllServe.mutex.synchronize do
+        running = JekyllServe.running?
+        JekyllServe.run_cond.wait(JekyllServe.mutex) unless running
       end
+    end
 
-      after(:each) do
-        JekyllServe.shutdown
+    after(:all) do
+      JekyllServe.shutdown
 
-        JekyllServe.mutex.synchronize do
-          running = JekyllServe.running?
-          JekyllServe.run_cond.wait(JekyllServe.mutex) if running
-        end
+      JekyllServe.mutex.synchronize do
+        running = JekyllServe.running?
+        JekyllServe.run_cond.wait(JekyllServe.mutex) if running
       end
+    end
+
+    describe 'weird file' do
+      data_dir = File.join(__dir__, '..', '..', '..', 'tests', 'full')
```

Comment:
Codacy found an issue: [Useless assignment to variable - `data_dir`.](https://app.codacy.com/gh/SwedbankPay/jekyll-plantuml-docker/pullRequest?prid=5820027)